src/nupic/regions/SPRegion.py

```diff
@@ -159,29 +159,25 @@ def getConstraints(arg):
       description='Total number of columns (coincidences).',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     inputWidth=dict(
       description='Size of inputs to the SP.',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     spInputNonZeros=dict(
       description='The indices of the non-zero inputs to the spatial pooler',
       accessMode='Read',
       dataType='UInt32',
-      count=0,
-      constraints=''),
```

Comment:
@vitaly-krugl 

I realized the mistep after the pull request. I will endeavor to issue a pull request after a commit, and try not to file several commits then issue a pull request. I've had other commits merged successfully. Thank you for your patience and guidance as i work through the learning curve.